man/liftoffrc.5

```diff
@@ -157,9 +157,9 @@ default:
 Specify the dependency managers to use.
 .Pp
 Currently, this accepts
-.Ic cocoapods
+.Ic cocoapods, carthage
```

Comment:
the extra space is so that it marks `cocoapods`, but not `,`.

PROTIP: you can preview this by passing a path to `man`:

```
man man/liftoffrc.5
```